ec/resources/waf/access_rule.go

```diff
@@ -457,18 +447,214 @@ func ResourceAccessRuleCreate(ctx context.Context, d *schema.ResourceData, m int
 	log.Printf("[INFO] Successfully created WAF Access Rule: %+v", resp)
 
 	d.SetId(resp.ID)
+	ResourceAccessRuleRead(ctx, d, m)
```

Comment:
We should probably combine the result of `ResourceAccessRuleRead` with the existing `diags` variable here. Otherwise, any errors from `ResourceAccessRuleRead` will be lost

Reply:
the RuleRead is not my concern here. I just copied the Dashan's code for work. We should address it Dashan's code.